include/dao.hpp

```diff
@@ -80,7 +80,8 @@ namespace hypha
       ACTION propose(const name &proposer, const name &proposal_type, ContentGroups &content_groups);
       ACTION vote(const name& voter, const checksum256 &proposal_hash, string &vote, string notes);
       ACTION closedocprop(const checksum256 &proposal_hash);
-      ACTION setsetting(const string &key, const Content::FlexValue &value);
+      // ACTION setsetting(const string &key, const Content::FlexValue &value);
+      ACTION setsetting(const eosio::name &dao_name, const std::string &key, const Content::FlexValue &value);
       ACTION remsetting(const string &key);
```

Comment:
Now that the contract will serve many DHOs, the dao_name needs to be provided as an input to each action.

The ```dao_name```, and corresponding hash, is the immutable unique identifier for the DHO. When the name is passed, the smart contract will generate a hash to find the document.  Once if finds this ```dho``` document, all other documents can be found hanging off of it. 

The ```dho``` doc look like this. 

```
ID            0
Hash          b9fdc8319323ecaae7812433c5e8b5e2ccd263f1c523c1cfdc7015d4f7165533
Creator       bm.hypha
Created Date  2021 Jun 18 22:43:04

ContentGroups
  ContentGroup
    content_group_label                details                                                          
    dao_name                           bm                                                               
  ContentGroup
    content_group_label                system                                                           
    type                               dao                                                              
    node_label                         bm             
```